examples/joke_bot/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "misterbot"
+version = "0.1.0"
+authors = ["Isaiah Inuwa <isaiah.inuwa@gmail.com>"]
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ruma = { version = "0.4.0", path = "../../crates/ruma", features = ["client-api-c", "client", "client-hyper-native-tls", "events"] }
+ruma-client = { version = "0.7.0", path = "../../crates/ruma-client", features = ["client-api"]}
+# For building locally: use the git dependencies below.
+# Browse the source at this revision here: https://github.com/ruma/ruma/tree/8923881678941a07ba941cdb4c44d5da06fde056
+# ruma = { git = "https://github.com/ruma/ruma", rev = "8923881678941a07ba941cdb4c44d5da06fde056", features = ["client-api-c", "client", "client-hyper-native-tls", "events"] }
+# ruma-client = { git = "https://github.com/ruma/ruma", rev = "8923881678941a07ba941cdb4c44d5da06fde056", features = ["client-api"] }
+
+http = "0.2.2"
+hyper = "0.14.2"
+hyper-tls = "0.5.0"
+serde_json = "1.0"
+tokio = { version = "1", features = ["full"] }
+tokio-stream = "0.1.7"
+
+[workspace]
```

Comment:
Why would this package have its own workspace?

Reply:
That was leftover from when I was not using a path dependency, and I used this to exclude this from the workspace so it would build.